_sass/pages/_docs.scss

```diff
@@ -36,4 +36,8 @@
     padding: 0;
     border-left: none;
   }
+
+  img {
+    width: 100%;
+  }
```

Comment:
The picture was overflowing outside the container: 
![image](https://cloud.githubusercontent.com/assets/1493671/21158888/5140ebfe-c17f-11e6-8ab7-df94e096227e.png)